src/store/data.ts

```diff
@@ -308,10 +332,15 @@ export const readLink = (
 export const writeRecord = (
   entityKey: string,
   fieldKey: string,
-  value: EntityField
+  value: EntityField,
+  skipPersistance?: boolean
```

Comment:
I opted for a `skipPersistance` boolean so we can skip writing as a whole when we are hydrating

Reply:
Hm I’m not sure about that. Wouldn’t it be better to just not have storage applied during that time?

Reply:
Hmm we could fake that by setting the adapter later I suppose but it feels quite implicit while this gives an explicit touch to that.

An explicit argument to skip is more obvious for new contributors than just setting storage.current at the end of `hydrateData`, up to you mainly my opinion